pkg/controller/Dockerfile

```diff
@@ -4,8 +4,8 @@ LABEL   description="Bare-metal CSI Controller Service"
 
 ADD     controller /controller
 
-RUN addgroup -S bmcsi && adduser -S bmcsi -G bmcsi
+RUN addgroup -S bmcsi && adduser -S bmcsi -u 1000 -G bmcsi
 
-USER bmcsi
+USER 1000
```

Comment:
This will set only user `UUID` and not `GID` why not use `bmcsi` user ?

Reply:
If you ask, why this changes are needed - to pass cis hardening validation. Without directly specifying UUID it would not work.

Reply:
I just dont get why you use `UID` instead of name.

Reply:
adduser -u 1000 creates a user with id 1000 
if you're using just `USER` directive so `RUN` directive about is useless, correct me if I'm wrong.

Reply:
There are some differences between this options. But more transparent way is to create and use the actual user.

Reply:
No, in this case cis validation will fail

Reply:
ok, got it